src/Client.php

```diff
@@ -33,6 +33,8 @@ public function __construct($parameters = null, $options = null)
         $profile->defineCommand('poolGet', 'PhpRQ\Command\Pool\Get');
         $profile->defineCommand('poolAck', 'PhpRQ\Command\Pool\Ack');
         $profile->defineCommand('poolRemove', 'PhpRQ\Command\Pool\Remove');
+
```

Comment:
se podívej nad to, je to rozdělené po sekcích :-) takže jo, je to "něco extra" :D